.env.example

```diff
@@ -50,6 +50,19 @@ GITHUB_TOKEN={fill me in}
 
 # ENV_WHITELIST={optional, comma separated list of env values that should be used when deploying }
 
+## EXPORT_JOB_DOWNLOADED_AGE determines how long a csv export job and file should
+## persist after it is downloaded before cleanup, default is 12.hours.
+##
+## EXPORT_JOB_MAX_AGE determines how long a csv export job and file should persist
```

Comment:
is there a usecase where you request a download and then come back a month later to download it ?
can we delete all downloads after 24 hours for example ?